tests/framework/clients/rancher/generated/management/v3/zz_generated_aks_imported_cluster_config_spec.go

```diff
@@ -0,0 +1,50 @@
+package client
+
+const (
```

Comment:
This file does not appear to be generated, did you implement this Markus? All files in clients/rancher/generated/ are generated using the framework/pkg/schemas. Because what will happen is there will be a v3 update, which is going to happen because of the updates in eks, this whole file will be deleted when we generate new structs.

Reply:
Correct, I had made this file to get aside from hardcoding values as we had seen earlier in the import test. That way, a user can just add to the config.yaml file to make it more dynamic, the import test that is.

Reply:
It doesn't belong here, all files in this directory are auto generated, as soon as we run go generate for updates this will be deleted. I guess I need more context to why this file is here. All these structs are for v3 API calls, is this an API call?

Reply:
Got it, makes sense. No API call, I had placed here as initially I believed I had to do that to allow for the import test to be made dynamic and not hard-coded. But it wounds that is done in `framework/pkg/schemas`. I will take a deeper look into that.